code/__DEFINES/preferences.dm

```diff
@@ -42,9 +42,10 @@
 #define SOUND_AMBIENCE (1<<2)
 #define SOUND_LOBBY (1<<3)
 #define SOUND_INSTRUMENTS_OFF (1<<4)
-#define SOUND_GAS_MASK (1<<5)
+#define SOUND_WEATHER (1<<5)
+#define SOUND_GAS_MASK (1<<6)
```

Comment:
do not change what is 1<<5 it will break already set prefs

Reply:
Fixed